script/railcar

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
```

Comment:
This comment was addressed, but then it popped back up after back after I rewrote the script. :unamused: